package.json

```diff
@@ -393,281 +393,281 @@
             ],
             "view/item/context": [
                 {
-                    "command": "azureFunctions.selectSubscriptions",
-                    "when": "view == azFuncTree && viewItem == azureextensionui.azureSubscription",
-                    "group": "inline"
+                    "command": "azureFunctions.addBinding",
```

Comment:
I'm not sure if we want to actually sort these alphabetically since they're currently (loosely) sorted by group and `viewItem`.

Reply:
Makes sense. I'll exclude this from the sort